<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTask</class>
 <widget class="QMainWindow" name="AddTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Task</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-image: url(:/images/background/backverticalback.png);
 }
QGroupBox {
border: 2px solid #aa55ff;
border-radius: 15px;
}
QCalendarWidget {
border-radius: 7px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>85</y>
      <width>250</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 15pt &quot;Merriweather&quot;;
</string>
    </property>
    <property name="title">
     <string>Add Task</string>
    </property>
    <widget class="QLabel" name="labelTaskName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>38</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Merriweather&quot;;</string>
     </property>
     <property name="text">
      <string>Task Name :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTaskName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>38</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
}
QLineEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelError">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>250</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Merriweather&quot;;
color: rgb(255, 0, 0);
qproperty-alignment: 'AlignHCenter';</string>
     </property>
     <property name="text">
      <string>error</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>72</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Merriweather&quot;;
image: url(:/images/icon/date_icon.png);
image-position: left;</string>
     </property>
     <property name="text">
      <string>       Date/Time :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDetails">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>248</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Merriweather&quot;;
image: url(:/images/icon/detail_icon.png);
image-position: left;</string>
     </property>
     <property name="text">
      <string>       Details :</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>103</y>
       <width>210</width>
       <height>130</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#calendarWidget QWidget {
	alternate-background-color: 	rgb(180, 135, 255);
	font: 5pt&quot;Merriweather&quot;;
	color: black;
}
#qt_calendar_navigationbar {
	background-color: rgb(215, 200, 255);
	border: 1px solid black;
	border-bottom: none;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
}
#qt_calendar_prevmonth,
#qt_calendar_nextmonth {
	border: none;
	qproperty-icon: none;
	border-radius: 7px;
	background-color: transparent;
	padding: 5px;
	icon-size: 10px 10px;
}
#qt_calendar_prevmonth {
	image: url(:/images/icon/arrow_left_icon.png);
}
#qt_calendar_nextmonth {
	image: url(:/images/icon/arrow_right_icon.png);
}
#qt_calendar_yearbutton {
	border-radius: 7px;
	padding: 5px 0;
}
#qt_calendar_monthbutton {
	border-radius: 7px;
	padding: 0 5px;
}
#qt_calendar_yearedit {
	background: transparent;
	width: 50px;
}
#qt_calendar_yearedit:down-button {
	image: url(:/images/icon/arrow_down_icon.png);
	subcontrol-position: right;
}
#qt_calendar_yearedit:up-button {
	image: url(:/images/icon/arrow_up_icon.png);
	subcontrol-position: left;
}
#qt_calendar_yearedit:down-button,
#qt_calendar_yearedit:up-button {
	width: 10px;
	height: 10px;
}
#qt_calendar_yearedit:down-button:hover,
#qt_calendar_yearedit:up-button:hover {
	background-color: rgb(199, 185, 236);
}
#qt_calendar_yearedit:down-button:pressed,
#qt_calendar_yearedit:up-button:pressed {
	background-color: rgb(179, 167, 213);
}
#calendarWidget QToolButton QMenu {
	background-color: rgb(215, 200, 255);
	border-radius: 3px;
	border: 1px solid black;
}
#calendarWidget QToolButton QMenu::item:selected:enabled {
	background-color: rgb(199, 185, 236);
}
#calendarWidget QToolButton::menu-indicator {
	nosubcontrol-origin: margin;
	subcontrol-position: right center;
	image: none;
}
#qt_calendar_calendarview {
	border: 1px solid black;

	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
}
#qt_calendar_calendarview::item:hover {
	background-color: rgb(199, 185, 236);
	border-radius: 7px;
}
#qt_calendar_calendarview::item:selected {
	background-color: rgb(179, 167, 213);
	border-radius: 7px;
}
#calendarWidget QAbstractItemView {
	background-color: rgb(215, 200, 255);
}</string>
     </property>
     <property name="gridVisible">
      <bool>false</bool>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
     <property name="navigationBarVisible">
      <bool>true</bool>
     </property>
     <property name="dateEditEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelAssign">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>303</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Merriweather&quot;;
image: url(:/images/icon/assign_icon.png);
image-position: left;</string>
     </property>
     <property name="text">
      <string>         Assign :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAssign">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
}
QLineEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditDetails">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
padding-top: 0px;
padding-bottom: 20px;
}
QTextEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>355</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover {
  background-color: rgb(165, 125, 235);
}
QPushButton {
	font: 10pt&quot;Merriweather&quot;;
	border-radius: 7px;
	background-color: rgb(180, 135, 255);
	border: 1px solid #000000;
	color: rgb(0, 0, 0);
}
QPushButton:pressed {
	background-color: rgb(135, 100, 195);
}</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStar">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>38</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icon/star_icon.png</normaloff>
       <normalon>:/images/icon/filled_star_icon.png</normalon>:/images/icon/star_icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>355</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover {
  background-color: rgb(165, 125, 235);
}
QPushButton {
	font: 10pt&quot;Merriweather&quot;;
	border-radius: 7px;
	background-color: rgb(180, 135, 255);
	border: 1px solid #000000;
	color: rgb(0, 0, 0);
}
QPushButton:pressed {
	background-color: rgb(135, 100, 195);
}</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
